web_src/js/features/user-auth-webauthn.js

```diff
@@ -85,14 +96,21 @@ function verifyAssertion(assertedCredential) {
   });
 }
 
-// Encode an ArrayBuffer into a base64 string.
-function bufferEncode(value) {
-  return encode(value)
+// Encode an ArrayBuffer into a URLEncoded base64 string.
+function bufferURLEncodedBase64(value) {
```

Comment:
Maybe it could be `encodeURLEncodedBase64` like `decodeURLEncodedBase64`

Reply:
done